### DISCUSSION TITLE
Calling peaks for many conditions

### DISCUSSION THREAD

**Discussion by EraslanBas** [GreenleafLab/ArchR#1882] [Questions / Documentation]
Hi,

I need to call peaks for about ~1000 different conditions, but I would like to get all the peaks for a given condition, not just the condition specific ones. When I use addReproduciblePeakSet() function I lose a lot of peaks that are common to certain conditions. May I ask what may be the most feasible solution to this problem?

Thank you.

**Comment by rcorces** [comment]
I'm not sure I understand your question. "all the peaks for a given condition, not just the condition specific ones" - these are the same thing in my mind.
